assembly/pom.xml

```diff
@@ -21,7 +21,7 @@
     <parent>
         <artifactId>spliceengine-parent</artifactId>
         <groupId>com.splicemachine</groupId>
-        <version>3.0.0.1944-SNAPSHOT</version>
+        <version>3.0.0.1945-SNAPSHOT</version>
```

Comment:
Not sure why these are a part of this PR.  Is this an intentional change?

Reply:
Thanks @carolp-503 I don't know how these changes get into this PR. To make it simple, I'm going to close this PR and open a new one.